cli/main.py

```diff
@@ -14,8 +14,8 @@ class bcolors:
 
 
 """
-To add a new deployment, add a new DeploymentName enum and a new entry in the DEPLOYMENTS dictionary containing the secrets required for the deployment.
-If the deployment requires a custom implementation, add a new key "custom_implementation" with the function that will be called to set up the deployment.
+To add a new model, create a new DeploymentName enum and a new entry in the DEPLOYMENTS dictionary containing the secrets required for the model deployment.
```

Comment:
Should we default to calling it `model deployment`? It might be confusing to refer to this by `model` or by `deployment` separately

Reply:
Agreed